include/swift/SIL/FormalLinkage.h

```diff
@@ -35,11 +35,7 @@ enum class FormalLinkage {
 
   /// This entity is visible in only a single Swift module and has a
   /// unique file that is known to define it.
-  HiddenUnique,
-
-  /// This entity is visible in only a single Swift module but does not
-  /// have a unique file that is known to define it.
-  HiddenNonUnique,
+  Hidden,
```

Comment:
Removing HiddenNonUnique makes sense to me if we never use it currently.  Renaming HiddenUnique seems like it doesn't actually win us anything, and it just creates pain if we ever need to reintroduce HiddenNonUnique.